### ISSUE TITLE
Implement the `ServiceInfo` endpoint

### ISSUE THREAD

**Issue by danielpetisme-bib** [osimloeff/PI-Web-API-Client-Angular#1]
Hello,

I want to get the Document Count attribute and I found the endpoint `<host>/piwebapi/search/serviceinfo` but I can find it in the endpoints.

Did I missed something?

**Comment by osimloeff** [comment]
This endpoint does not exist. Please visit the page below to find out about the controllers and methods available: 

https://techsupport.osisoft.com/Documentation/PI-Web-API/help.html

Hope this helps!

**Comment by danielpetisme-bib** [comment]
Thank for the reactivity.
The endpoint does exist :wink: 
https://techsupport.osisoft.com/Documentation/PI-Web-API/help/controllers/search/actions/indexinfo.html

This is what I have on my instance
![image](https://user-images.githubusercontent.com/16398469/37429374-515d74d8-27cf-11e8-9d97-26eecabf8e06.png)

![image](https://user-images.githubusercontent.com/16398469/37429258-15f64d20-27cf-11e8-9d35-efbb179e84f6.png)

Some version info if it helps you
```
{
  "OSIsoft.REST": {
    "FullVersion": "1.9.0.478",
    "MajorMinorRevision": "1.9.0",
    "Build": "478"
  },
  "OSIsoft.REST.Documentation": {
    "FullVersion": "1.9.0.478",
    "MajorMinorRevision": "1.9.0",
    "Build": "478"
  }  
...
}
```

**Comment by osimloeff** [comment]
Ok, so the Search controller was not added to the Swagger specification. This is why all client libraries does not support API calls against the search controller. The only option is if you download the source code, add the search controller and compile again. You can make HTTP requests without using the client library as well.

**Comment by danielpetisme-bib** [comment]
Indeed, I have to check how to use to use Kerberos via Angular http client.
Maybe we can raise a cas on Osisoft side ?

Thank you.  for you help I think we can close the case ?

**Comment by osimloeff** [comment]
Yes, you can use Kerberos. Just read the README.MD file again. There is an example.
